test/CodeGeneration/CodeGenerator.Tests/CodeGenerator.Tests.csproj

```diff
@@ -4,11 +4,14 @@
     <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
     <IsPackable>false</IsPackable>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <OrleansBuildTimeCodeGen>true</OrleansBuildTimeCodeGen>
```

Comment:
Even though I'm not using the generated .g.cs file (i'm emulating the generated call site instead), I want to put codegen to work against the interface being used for testing. A canary, of sorts.